src/index.ts

```diff
@@ -41,6 +51,51 @@ export interface KeyStore {
     clear(): Promise<void>;
 }
 
+export interface HDWKeyStore {
+    getSeedHashes(): Promise<SeedHash[]>;
+    importSeed(params: {
+        secret: SecretSeedStorage;
+        passphrase?: string;
+        meta?: string;
+    }): Promise<SeedHash>;
+    importRawSeed(params: {
+        seed: Seed;
+        passphrase?: string;
+        meta?: string;
```

Comment:
Where can we get `meta` data?
Do we need `getMeta` function in HDWKeyStore interface?

Reply:
Added.